data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,436 @@
+			`	"I think we've generally had enough of this here. Answer me two questions, and then we shall make our decision."`
+			`	"Do you think that a diplomatic post should be able to defend itself? Or should it trust to the strength of those who meet there to ensure its protection?"`
+			`	This seems like a very strange question...`
+			choice
+				`	"It should be strong enough to defend itself. You cannot trust your enemy."`
+					goto trust
+				`	"Both sides must be trusted to protect a diplomatic post, or there is no point to having one."`
+					goto caution
+			label trust
+			apply
+				set haitrust
+			`	You seem to get no reaction and the Elder follows up with, "What is the gravest threat, in your opinion, facing humanity?"`
+				goto return
+			label caution
+			apply
+				set haicaution
+			`	You seem to get no reaction and the Elder follows up with, "What is the gravest threat, in your opinion, facing humanity?"`
+			label return
+			choice
+				`	"The Pug"`
+				`	"The Alphas"`
+				`	"Unknown Jump-Capable Aliens"`
+				`	"Experimental Technologies"`
+				`	"Hubris"`
+					goto wise
+			`	There is a collection of chittering laughs from the various council members. You're not sure if there was something wrong with what you said but you aren't left long to ponder.`
+			`	"It seems the human agent is an honest fool, but an honest fool can at least be trusted to be foolish honestly on all fronts."`
+			`	You're pretty sure you should be offended by that, but the council members are smiling and don't look like they're about to evict you.`
+				goto end
+			label wise
+			`	"It seems you learnt the correct lesson from the crisis of your species," he says.`
+			`	You see several appreciative nods happening around the room out of the corner of your eyes.`
+			label end
+			`	"You may act as the oversight for the human governments in this project. Proposal D-86513-7P4 is approved. Council adjourned."`
+			`	When you meet Sayari afterwards a touch of her usual enthusiasm returns as you share the news.`
+			`	"This is fantastic news, now all that remains to be done is for you to escort the construction fleet from Makerplace to Ultima Thule and observe the initial stages. They'll be waiting for your arrival on Makerplace."`
+				accept
+	on accept
+		event "role-play station before construction"
+		log `The diplomats have elected to double-down on the role-play community angle, citing its success. However, the Hai are unwilling to trust a human government to see the inner workings of a station under construction in the wake of the civil war, and the Republic will not allow a station to be built on their side unsupervised. You have been asked to fulfill the role of supervisor as a result. The Elders asked you some odd questions, but agreed that you were an acceptable individual for the task.`
```

Comment:
```suggestion
		log `The diplomats have elected to double-down on the role-play community angle, citing its success. The Hai, however, are unwilling to trust a human government to see the inner workings of a station under construction in the wake of the civil war, and the Republic will not allow a station to be built on their side unsupervised. You have been asked to fulfill the role of supervisor as a result. The Elders asked you some odd questions, but agreed that you were an acceptable individual for the task.`
```
I learned as a stylistic rule to never start a sentence with "however" if you can avoid it, but up to you.

Reply:
Should be a ; before "however" then.

Reply:
The phrase has to be a complete sentence before a semicolon. I think there's an exception to the rule of thumb where if the `however` at the start feels absolutely necessary, you use a semicolon instead of a period at the end of the previous sentence. Are you thinking of that? In that case, it'd look like this:
```suggestion
		log `The diplomats have elected to double-down on the role-play community angle, citing its success; however, the Hai are unwilling to trust a human government to see the inner workings of a station under construction in the wake of the civil war, and the Republic will not allow a station to be built on their side unsupervised. You have been asked to fulfill the role of supervisor as a result. The Elders asked you some odd questions, but agreed that you were an acceptable individual for the task.`
```